src/app/seamly2d/mainwindow.cpp

```diff
@@ -977,20 +977,20 @@ void MainWindow::handleTriangleTool(bool checked)
 
 //---------------------------------------------------------------------------------------------------------------------
 /**
- * @brief handlePointOfIntersectionTool handler for pointOfIntersection tool.
+ * @brief handlePointIntersectXyTool handler for pointOfIntersection tool.
```

Comment:
For handlePointIntersectXyTool, why isn't Y capitalized? This would be consistent with your new naming convention of PointIntersectXYTool, etc.